src/main/resources/assets/sandwichable/lang/fr_fr.json

```diff
@@ -0,0 +1,283 @@
+	"block.sandwichable.basalt_basin.info": "Bloc utilisé pour faire du fromage. Pour cela, interagir dessus avec un seau de lait, puis avec une fiole de cultures pour fromage pour commencer le processus de fermentation. Au bout de trois minutes, des particules apparaissent. Interagir pour retirer le fromage. Le Lait fermenté se récupère avec un seau.",
+	"block.sandwichable.blackstone_basin.info": "Bloc utilisé pour faire du fromage. Pour cela, interagir dessus avec un seau de lait, puis avec une fiole de cultures pour fromage pour commencer le processus de fermentation. Au bout de trois minutes, des particules apparaissent. Interagir pour retirer le fromage. Le Lait fermenté se récupère avec un seau.",
+	"block.sandwichable.bottle_crate.info": "Bloc de stockage pour les fioles de cultures pour fromage. Une fois rangées, les fioles vont lentement s'affiner, augmentant  la quatité de cultures pour fromage à l'interieur. Utilisé pour cultiver les cultures pour fromage. La caisse peut être ouverte pour en exhiber les fioles en interagissant dessus en étant accroupi.",
+	"block.sandwichable.shrub.info": "Petite plante que l'on peut trouver partout dans le monde. Elle peut être détruite normalement pour faire tomber des graines de nouvelles plantes, ou détruite avec des cisailles pour obtenir l'arbuste lui-même. On peut également la tailler pour en retirer les feuilles ou la mettre en pot. Mourra si placée sur du sable ou toastée.",
+	"block.sandwichable.salty_sand.info": "Bloc minéral que l'on trouve sur les plages et dans les océans, peut être détruit pour obtenir une pierre de sel et un peu de sable.",
+	"block.sandwichable.salty_rocks.info": "Bloc ne pouvant pas être récupéré, se trouvant sur les bords d'étendues d'eau salée générées naturellement. L'eau au dessus de celui-ci sera salée, ce qui signifie qu'un dessalinisateur fonctionnera en étant posé sur ce bloc.",
+	"block.sandwichable.salty_stone.info": "Bloc ne pouvant pas être récupéré, se trouvant dans les fonds d'étendues d'eau salée générées naturellement. L'eau au dessus de celui-ci sera salée, ce qui signifie qu'un dessalinisateur fonctionnera en étant posé sur ce bloc.",
+	"block.sandwichable.desalinator.info": "Machine pour extraire le sel des plages et de l'eau de mer. Contrairement au toasteur à redstone, il est alimenté uniquement par de la poudre de redstone, non une impulsion. Son réservoir se remplit avec de l'eau qui donnera du sel si elle se trouve dans un biome salé."
+}
+
+    © 2021 GitHub, Inc.
+    Terms
+    Privacy
+    Security
+    Status
+    Docs
+
+    Contact GitHub
+    Pricing
+    API
+    Training
+    Blog
+    About
```

Comment:
This appears to be a mistake where too much was copied when copying a template for the lang file. As it is the footer of GitHub.
```suggestion
```

Reply:
So I need to remove this footer and it will be okay ? (Sorry, real noob here...)

Reply:
Yes, at least the file. I have not verified the actual translations since I don't speak french. Use [en_us.json](https://github.com/FoundationGames/Sandwichable/blob/v1.1/src/main/resources/assets/sandwichable/lang/en_us.json) for reference. The closing curly bracket should be the last line of the file.